modules/template/template.go

```diff
@@ -104,6 +105,15 @@ func NewFuncMap() []template.FuncMap {
 		"ThemeColorMetaTag": func() string {
 			return setting.ThemeColorMetaTag
 		},
+		"TabSizeClass": func(ec *editorconfig.Editorconfig, filename string) string {
+			if ec != nil {
+				def := ec.GetDefinitionForFilename(filename)
+				if def.TabWidth > 0 {
+					return fmt.Sprintf("tab-size-%d", def.TabWidth)
+				}
+			}
+			return "tab-size-8"
```

Comment:
I will let this open for discussion, but in theory 8 is the default for tabs.

What we can do is allowing one change the default in `app.ini`, as well disable the `editorconfig` functionality all together. I think it would be a good idea, in case someone dislike this. What do you think?